anomalib/models/components/feature_extractors/torchfx.py

```diff
@@ -0,0 +1,52 @@
+"""Feature Extractor based on TrochFX."""
+
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import importlib
+from typing import List, Optional
+
+from torch.fx.graph_module import GraphModule
+from torchvision.models._api import WeightsEnum
+from torchvision.models.feature_extraction import create_feature_extractor
+
+
+def get_torchfx_feature_extractor(
```

Comment:
Do we really need this as a separate function? We could just call `create_feature_extractor` directly from the code right?

Reply:
We also have to freeze the gradients. We might miss doing this in the models. This provides an interface similar to our existing implementation. But instead of `pretrained` we have `weights` now as pretrained is going to be deprecated in torchvision.

Reply:
I see. In that case this is a good solution. I'm surprised torchvision does not freeze the gradients by default in `create_feature_extractor`, if it reduces memory use.